app/views/wpcc/shared/_your_details.html.erb

```diff
@@ -11,4 +11,11 @@
       <%= link_to(t('wpcc.edit'), new_your_detail_path, class: 'section__heading-edit') %>
     </span>
   </h2>
+  <% if session[:manually_opt_in] %>
```

Comment:
The same presenter that will handle Jone's PR should be doing this check. I am not sure whuch wll go first. There on Jones'Pr or here.